SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/www/js/index.js

```diff
@@ -0,0 +1,114 @@
+var oauth = new SalesforceJS.OAuth2();
+
+var fetchUsers = function () {
+    var accounts;
+    var account;
+
+    oauth.getUser(function success(result) {
+        account = result;
+    });
+    oauth.getUsers(function success(result) {
+        accounts = result;
+    });
+    var buttonli;
+    var buttondiv;
+    var userButton;
+    var listItemsHtml = document.querySelector('#users');
+    if (accounts != null) {
+        for (var x = 0; x < accounts.length; x++) {
+            var current = accounts[x];
+            if (current.userId == account.userId) {
+                var li = document.createElement("li");
+                var div = document.createElement("div");
+
+                li.setAttribute("class", "table-view-cell");
+                div.setAttribute("class", "media-body");
+
+                div.innerHTML = current.userName;
+                li.appendChild(div);
+                listItemsHtml.appendChild(li);
+            } else {
+                buttonli = document.createElement("li");
+                buttondiv = document.createElement("div");
+                userButton = document.createElement("button");
+                buttonli.setAttribute("class", "table-view-cell");
+                buttonli.setAttribute("align", "center");
+                buttondiv.setAttribute("class", "media-body");
+                userButton.addEventListener("click", userItem(current), false);
+                userButton.innerText = current.userName;
+                buttondiv.appendChild(userButton);
+                buttonli.appendChild(buttondiv);
+                listItemsHtml.appendChild(buttonli);
+            }
+        }
+        
+    }
+}
+
+var userItem = function (account) {
+    return function () { switchUser(account); }
+}
+
+var fetchContacts = function () {
+    var soql = 'SELECT Id, Name FROM Contact LIMIT 10';
+    var rest = Salesforce.SDK.Hybrid.Rest;
+    var cm = new rest.ClientManager();
+    var client = cm.peekRestClient();
+
+    var request = rest.RestRequest.getRequestForQuery("v31.0", soql);
```

Comment:
Let's replace ```v31.0``` with a variable for API version if possible. It makes it easier to bump up the API versions with releases.